src/components/Deliverable/DeliverableTrackline.tsx

```diff
@@ -128,9 +127,13 @@ function DeliverableTrackLine(props: DeliverableTargetLineProps) {
 					donors={donors}
 					TracklineId={data.updateDeliverableTrackingLineitemDetail.id}
 					TracklineFyId={data.updateDeliverableTrackingLineitemDetail.financial_year?.id}
-					data={donorFormData}
+					data={Object.keys(donorFormData).length ? donorFormData : {}}
 					onCancel={onCancel}
-					type={FORM_ACTIONS.UPDATE}
+					type={
+						Object.keys(donorFormData).length
```

Comment:
there  should be a better way to do this

Reply:
it stores dynamic key fields and its not required at the time of creation thats why i use keys length